script/generate-metadata-for-builder.js

```diff
@@ -0,0 +1,209 @@
+'use strict';
+
+const CSON = require('season');
+const fs = require('fs-plus');
+const normalizePackageData = require('normalize-package-data');
+const path = require('path');
+const semver = require('semver');
+
+module.exports = function(packageJSON) {
+  return {
+    _atomPackages: buildBundledPackagesMetadata(packageJSON),
+    _atomMenu: buildPlatformMenuMetadata(packageJSON),
+    _atomKeymaps: buildPlatformKeymapsMetadata(packageJSON)
```

Comment:
Why do the properties have `_` prefixes? This is usually indicative of unused properties.